package.json

```
@@ -4,7 +4,7 @@
   "version": "1.47.1",
   "description": "A framework to support application development with Terra components",
   "engines": {
-    "node": ">=8.9.2 <11"
+    "node": ">=10"
```

I think you have to specify less than 16 to build on heroku
```suggestion
    "node": ">=10 <16"
```